scripts/install_godot_templates_linux.rb

```diff
@@ -7,7 +7,7 @@
 require_relative 'bootstrap_rubysetupsystem'
 require_relative 'RubySetupSystem/RubyCommon'
```

Comment:
This script probably blows up here.
You should actually test these scripts to make sure you don't break things.